build/Dockerfile

```diff
@@ -69,4 +72,4 @@ LABEL \
       maintainer="Roddie Kieley <rkieley@redhat.com>"  \
       name="amq7/amq-broker-rhel7-operator" \
       summary="ActiveMQ Artemis Broker Operator"  \
-      version="0.17"
+      version="0.18"
```

Comment:
JBOSS_IMAGE_VERSION="0.17" in build/Dockerfile

Reply:
@RoddieKieley not sure what's the label for in upstream. any idea if it's necessary?

Reply:
@gaohoward I suspect that at this point the label is not required. But we can review and be sure before removal via a clean-up pass instead of mixing with the init container work.

Reply:
Ok I'll just remove it and keep it in mind when something wrong happens.